package.json

```diff
@@ -58,7 +58,7 @@
     }
   },
   "lint-staged": {
-    "*.{ts,tsx,js,jsx,css,json,yaml,md}": [
+    "*.{ts,tsx,js,jsx,css,json,md}": [
```

Comment:
Is this a temporary change?

Reply:
no, permanent - we don't need prettier to be modifying yaml files (and we don't actually commit that many)

Reply:
I think he did this to prevent the helm templates being wrecked by prettier / tslint.